android/app/build.gradle

```diff
@@ -256,7 +256,7 @@ dependencies {
     implementation project(':react-native-keychain')
     implementation project(':instabug-reactnative')
 
-    implementation 'status-im:function:0.0.1'
+    compile 'com.github.status-im:function:0.0.1'
```

Comment:
is there an explanation for this ?

Reply:
jitpack is used instead of custom storage for `jar` in this case. So literally dependency is "loaded" from github. 
https://jitpack.io/